### PR TITLE
Problem Suggestion: Add "Tree Compass" to Silver - Additional Practice for USACO Silver

### PR DESCRIPTION
**Pull request by maggieliu05**
User `IwlvchNg6zeZVoG3Pl0r8OOiq3f2` suggested adding the problem [Tree Compass](https://codeforces.com/contest/1943/problem/C) to the `problems` table of the module [Silver - Additional Practice for USACO Silver](https://usaco.guide/silver/silver-conclusion#problemlist-problems).

**Automatically Generated JSON:**
```json
{
  "uniqueId": "cf-1943C",
  "name": "Tree Compass",
  "url": "https://codeforces.com/contest/1943/problem/C",
  "source": "CF",
  "difficulty": "Normal",
  "isStarred": false,
  "tags": [
    "Tree"
  ],
  "solutionMetadata": {
    "kind": "autogen-label-from-site",
    "site": "CF"
  }
}
```
**Additional Notes**:
https://codeforces.com/blog/entry/127195

*This PR was automatically generated from a user-submitted problem suggestion on the USACO guide.*

### CODE REVIEW

**Comment by SansPapyrus683** [comment]
@envyaims go

**Comment by envyaims** [comment]
put this in Gold Hard

File: content/3_Silver/Conclusion.problems.json [hunk 1]
```diff
@@ -241,6 +241,19 @@
         "site": "CF"
       }
     },
+    {
+      "uniqueId": "cf-1943C",
+      "name": "Tree Compass",
+      "url": "https://codeforces.com/contest/1943/problem/C",
+      "source": "CF",
+      "difficulty": "Normal",
```
Comment (envyaims) [inline content/3_Silver/Conclusion.problems.json:249]:
```suggestion
      "difficulty": "Very Hard",
```
difficulty adjustment

Comment (SansPapyrus683) [reply content/3_Silver/Conclusion.problems.json:249]: 太阳从西边出来了

**Review by SansPapyrus683 — APPROVED** [review]

**Comment by SansPapyrus683** [comment]
image

is this still relevant

**Comment by envyaims** [comment]
> is this still relevant

No, I found an easier solution

**Review by envyaims — APPROVED** [review]

**Pull request merged**